src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1424,13 +1424,18 @@ public partial class StreamWriter : System.IO.TextWriter
         public override void Write(char value) { }
         public override void Write(char[] buffer) { }
         public override void Write(char[] buffer, int index, int count) { }
+        public override void Write(System.ReadOnlySpan<char> buffer) { }
         public override void Write(string value) { }
+        public override void WriteLine(string value) { }
+        public override void WriteLine(System.ReadOnlySpan<char> value) { }
```

Comment:
Should the arg name be `buffer` instead of `value`? Other methods in StreamWriter seem to have `value` for a single char and `buffer` for an array of chars.

Reply:
It should be `buffer`.